DesktopReact/frontend/src/components/PostFooter/PostFooter.js

```diff
@@ -24,89 +26,66 @@ const StyledBadge = styled(Badge)(({theme}) => ({
     },
 }));
 
-const PostFooter = () => {
-    const [isCollapse, setIsCollapse] = useState(false) // todo comment collapse function to listen the onClick
-    const [isToggled, setIsToggled] = useState(); // TODO handle the reaction icons, if one pressed set it true and the others to false.
+const PostFooter = (props) => {
+    //console.log(props)
+    //const [isCollapse, setIsCollapse] = useState(false) // todo comment collapse function to listen the onClick
+    //const [isToggled, setIsToggled] = useState(); // TODO handle the reaction icons, if one pressed set it true and the others to false.
+    const [onHover, setOnHover] = useState(false)
+    const onHoverMouse = () => { // added onHover func to comment button, to hide it on reaction overmouse. Needed?
+        setOnHover(!onHover)
+    }
+
     const dialStyle = {
         backgroundColor: "white", 
         minHeight: "2rem", 
-        height: "2rem", 
+        height: "2rem",
         width: "2rem",
         position: "relative",
-        boxShadow: "none"
+        boxShadow: "none",
+        margin: "2px",
     }
     const footerButtonStyle = {
         ...dialStyle,
         minWidth: "2rem",
         backgroundColor:'transparent'
     }
-    let badgeCounterValue = 1; // todo hardcoded values
-    // const actionSpeedDialNav = [
-    //     {id: 1, name: 'Love', icon: <FavoriteBorder/>, badgeCounter: badgeCounterValue},
-    //     {id: 2, name: 'Haha', icon: <Mood/>, badgeCounter: badgeCounterValue},
-    //     {id: 3, name: 'Wow', icon: <OutletOutlined/>, badgeCounter: badgeCounterValue},
-    //     {id: 4, name: 'Sad', icon: <SentimentVeryDissatisfied/>, badgeCounter: badgeCounterValue},
-    //     {id: 5, name: 'Angry', icon: <SentimentVeryDissatisfiedOutlined/>, badgeCounter: badgeCounterValue},
-    //     {id: 6, name: 'Care', icon: <AccessibilityNewOutlined/>, badgeCounter: badgeCounterValue},
-    // ];
+    //let badgeCounterValue = 1; // todo hardcoded values
+
+    // gets values from httpService.js array reactToPost function
+    const [badgeValues, setBadgeValues] = useState([])
+    const [likeValue, setLikeValue] = useState()
+
+    const BadgeCounterValue = () => {
+        let {data, result} = reactToPost();
+        //console.log(data);
+        setLikeValue(data[0][1])
```

Comment:
if modifying the same value ex. likeValue++ use a callback
setLikeValue(value => value++) to modify it

Reply:
Just needed the first value of the array. Still applies